src/tools/schema2cpp.ts

```diff
@@ -57,6 +57,24 @@ export class Schema2Cpp extends Schema2Base {
     return baseName.toLowerCase().replace(/\.arcs$/, '').replace(/[._]/g, '-') + '.h';
   }
 
+  generateEntityClassName(node: SchemaNode, i: number) {
+    if (i === -1) {
```

Comment:
Prefer using a named constant for this, or use null and make `i` have that as a default.

Reply:
Using null as default.